src/features/contactmoment/service.ts

```diff
@@ -31,19 +32,38 @@ const zaaksysteemApiRoot = `/zaken/api/v1`;
 const zaaksysteemBaseUri = `${zaaksysteemProxyRoot}${zaaksysteemApiRoot}`;
 const zaakcontactmomentUrl = `${zaaksysteemBaseUri}/zaakcontactmomenten`;
 
-export const saveContactmoment = (
+
+
+
+export const saveContactmoment = async (
```

Comment:
Is dit voor een andere story? Alleen ter info, deze comment mag je resolven

Reply:
bug fix.